src/UI/Command/CleanEvents.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Adshares\AdSelect\UI\Command;
+
+use Adshares\AdSelect\Application\Service\DataCleaner;
+use DateTime;
+use Exception;
+use function sprintf;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CleanEvents extends Command
+{
+    use CleanTrait;
+
+    protected static $defaultName = 'ops:es:clean-events';
+
+    /** @var DataCleaner */
+    private $dataCleaner;
+
+    public function __construct(DataCleaner $dataCleaner)
+    {
+        parent::__construct();
+        $this->dataCleaner = $dataCleaner;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Remove events from documents older than given date (or -2 days by default).')
+            ->addOption(
+                'from',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Older documents than `from` date will be removed. Supported format: Y-m-d H:i:s',
+                new DateTime('-10 days')
```

Comment:
-10 days is not equal -2 days. Compare lines 34 and 40.

Reply:
Fixed